test/index.js

```diff
@@ -67,4 +68,16 @@ describe('Sitemap generator', () => {
       result.should.be.ok;
     });
   });
+
+  it('IDN handling', () => {
+    hexo.config.url = 'http://fôo.com/bár';
+    const parsedUrl = encodeURL(hexo.config.url);
+
+    const result = generator(locals);
+    const $ = cheerio.load(result.data);
```

Comment:
`result.data` still show "http://yoursite.com/". Same issue as https://github.com/hexojs/hexo-generator-sitemap/pull/66/files#r314965773

Reply:
#70 resolve this issue, but brings another issue.

Reply:
Resolved